Documentation/ProjectFile/prj/processes/process/HEAT_TRANSPORT_BHE/borehole_heat_exchangers/borehole_heat_exchanger/flow_and_temperature_control/PowerCurveConstantFlow/t_power_curve.md

```diff
@@ -0,0 +1 @@
+It is the curve stands for the variation of heat load on the BHE.
```

Comment:
Suggestion:
~~It is t~~The curve stands for the variation of heat load on the BHE.

Reply:
Thanks for your comments. Fixed.